models/prices/prices_usd_latest_old.sql

```diff
@@ -0,0 +1,23 @@
+{{ config(
+        schema='prices',
+        alias = alias('usd_latest_old'),
+        tags= ['static', 'dunesql'],
```

Comment:
if we're doing incremental, i would remove `static` here

Reply:
Updated.